docker/pypi/wmagent/Dockerfile

```diff
@@ -1,14 +1,69 @@
-FROM registry.cern.ch/cmsweb/dmwm-base:pypi-20230314
+FROM registry.cern.ch/cmsweb/oracle:21_5-stable as oracle
+FROM registry.cern.ch/cmsweb/dmwm-base:pypi-20230512
 MAINTAINER Valentin Kuznetsov vkuznet@gmail.com
+
+# Install basic OS package dependencies
 RUN apt-get update
-RUN apt-get install -y libmariadb-dev-compat libmariadb-dev apache2-utils sudo
-ENV TAG=X.Y.Z
-RUN pip install wmagent==$TAG
-ENV WDIR=/data
-ENV USER=_wmagent
-RUN useradd ${USER} && install -o ${USER} -d ${WDIR}
-RUN echo "%$USER ALL=(ALL) NOPASSWD:ALL" >> /etc/sudoers
-USER ${USER}
-RUN sudo chown -R $USER.$USER $WDIR
-WORKDIR $WDIR
-CMD ["python3"]
+RUN apt-get install -y libmariadb-dev-compat libmariadb-dev apache2-utils hostname net-tools iputils-ping cron mariadb-server myproxy voms-clients rpm2cpio rlwrap libaio1 procps
```

Comment:
I strongly oppose this suggestion! This is potentially dangerous. Hope you meant:
`... && rm -rf /var/cache/apt/archives/*`, because this is where the temporary and cache files are kept  and could be left behind. And the proper command for cleaning those is not `rm -rf` but rather `apt-get clean`. And honestly the current size of it is 0:
```
(WMAgent-2.2.1) [cmst1@unit01:current]$ sudo  du -hs /var/cache/apt/archives/
0	/var/cache/apt/archives/
```

As of your suggestion: 
```
/var/lib/apt/lists/
           Storage area for state information for each package resource specified in sources.list(5) Configuration Item: Dir::State::Lists.
``` 
So if we do `rm -rf` there, it means the package lists would simply be wiped out. That won't hurt, until somebody logs into the container and decides there is a small missing package (possibly for speciffic debugging purposes), which he quickly tries to install by `apt-get install <pkgName>` but he skipped the step `apt-get update` beforehand, because he has forgotten that during build time we have wiped out all the lists, containing the full package dependency tree. Te result would the following:

```
(WMAgent-2.2.1) [cmst1@unit01:current]$ sudo rm -rf /var/lib/apt/lists/* 
(WMAgent-2.2.1) [cmst1@unit01:current]$ sudo apt-get install emacs-nox
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
E: Unable to locate package emacs-nox
```
So I am strongly against forcefully breaking the the packaging system inside the cntainer under the excuse.... "People are not supposed to install things in a running agent, so they wouldn't need the packaging system."

I know, if one remembers what we have done during the docker image creation time, he may fix things by running 
`apt-get update` and then `apt-get install`, but this means we just put one more future obstacle in our way. I hate remember things forever. One may say, I am a person of the moment ;) 

At the end we wont gain much:
```
(WMAgent-2.2.1) [cmst1@unit01:current]$ sudo  du -hs /var/lib/apt/lists/
18M	/var/lib/apt/lists/
```

Reply:
agreed on that part.

Reply:
done